swap/src/cli/list_sellers.rs

```diff
@@ -164,32 +165,32 @@ impl EventLoop {
                                 self.reachable_asb_address.insert(peer_id, address.clone());
                             }
                         }
-                        SwarmEvent::UnreachableAddr { peer_id, error, address, .. } => {
-                            if address == self.rendezvous_addr {
+                        SwarmEvent::OutgoingConnectionError { peer_id, error } => {
+                            if let Some(peer_id_rendevous) = peer_id {
                                 tracing::error!(
                                     "Failed to connect to rendezvous point at {}: {}",
-                                    address,
+                                    self.rendezvous_addr,
                                     error
                                 );
 
-                                // if the rendezvous node is unreachable we just stop
-                                return Vec::new();
-                            } else {
-                                tracing::debug!(
-                                    "Failed to connect to peer at {}: {}",
-                                    address,
-                                    error
-                                );
-                                self.unreachable_asb_address.insert(peer_id, address.clone());
+                                self.unreachable_asb_address.insert(peer_id_rendevous, self.rendezvous_addr.clone());
 
-                                match self.asb_quote_status.entry(peer_id) {
+                                match self.asb_quote_status.entry(peer_id_rendevous) {
                                     Entry::Occupied(mut entry) => {
                                         entry.insert(QuoteStatus::Received(Status::Unreachable));
                                     },
                                     _ => {
-                                        tracing::debug!(%peer_id, %error, "Connection error with unexpected peer")
+                                        tracing::debug!(%peer_id_rendevous, %error, "Connection error with unexpected peer")
                                     }
                                 }
+                            } else {
+                                tracing::error!(
+                                    "Failed to connect to rendezvous point at {}: {}",
```

Comment:
This should be "Failed to connect to peer at {}: {}". The peer is not the rendezvous point

Reply:
Did I make a mistake here? Aren't you checking if its the rendezvous point in line 169 and if its not you will print this message?

Reply:
Agreed with you and fixed in 7838442947c471ce6deb91f19e989bd636f6f076

Reply:
Thanks!